bondpy/bondpy/bondpy.py

```diff
@@ -335,18 +338,18 @@ def wait_until_broken(self, timeout=None):
             self.deadline = Timeout(timeout).reset()
         with self.lock:
             while self.sm.getState().getName() != 'SM.Dead':
-                if rospy.is_shutdown():
+                if not rclpy.ok(context=self.node.context):
                     break
-                if self.deadline and self.deadline.left() == rospy.Duration(0):
+                if self.deadline and self.deadline.left() == Duration(seconds=0):
                     break
                 wait_duration = 0.1
                 if self.deadline:
-                    wait_duration = min(wait_duration, self.deadline.left().to_sec())
+                    wait_duration = min(wait_duration, duration_to_sec(self.deadline.left()))
                 self.condition.wait(wait_duration)
             return self.sm.getState().getName() == 'SM.Dead'
 
     ## \brief Indicates if the bond is broken
-    # \return true iff the bond has been broken.
+    # \return true if the bond has been broken
```

Comment:
```suggestion
    # \return true iff the bond has been broken
```

Reply:
Changed in [74805b1](https://github.com/ros/bond_core/pull/89/commits/74805b1f61bde433b7f39eaf3db5ecf52190c80f)